src/app/(post)/result/[postId]/_components/CommentList/CommentItem/CommentItemMenu/index.tsx

```diff
@@ -0,0 +1,40 @@
+import { PropsWithChildren } from 'react';
+
+import * as DropdownMenu from '@radix-ui/react-dropdown-menu';
+
+import Icon from '@/src/components/Icon';
+
+const CommentItemMenu = ({ children }: PropsWithChildren) => {
+  return (
+    <DropdownMenu.Root>
+      <DropdownMenu.Trigger asChild>
+        <button>
+          <Icon id='more-square' />
+        </button>
+      </DropdownMenu.Trigger>
+      <DropdownMenu.Portal>
+        <DropdownMenu.Content
+          onCloseAutoFocus={e => e.preventDefault()}
+          className=' font-text-3-rg flex flex-col gap-2 rounded-lg bg-[#2d3033] p-2 text-white'
+        >
+          {children}
+        </DropdownMenu.Content>
+      </DropdownMenu.Portal>
+    </DropdownMenu.Root>
+  );
+};
+
+interface CommentItemMenuButtonProps {
+  label: string;
+  onClick: () => void;
+}
+
+const CommentItemMenuButton = ({
+  label,
+  onClick,
+}: CommentItemMenuButtonProps) => {
+  return <DropdownMenu.Item onSelect={onClick}>{label}</DropdownMenu.Item>;
+};
```

Comment:
```suggestion
const CommentItem = ({
  label,
  onSelect,
}: CommentItemMenuButtonProps) => {
  return <DropdownMenu.Item onSelect={onSelect}>{label}</DropdownMenu.Item>;
};
```

기존에 Item을 그대로 사용해도 좋을 것 같은데 Button으로 변경한 이유가 있을까요 ? 와 크게 다르지 않은 것 같아서 여쭤봅니다 !

Reply:
아니요 없습니다.! 아마 DropdownMenu.Item 이것과 겹친다 생각해서 Button 으로 수정한것 같습니다! 

수정해보겠습니다!